src/Symfony/Component/Validator/Constraints/NotPwned.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Checks if a password has been leaked in a data breach.
+ *
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class NotPwned extends Constraint
+{
+    const PWNED_ERROR = 'd9bcdbfe-a9d6-4bfa-a8ff-da5fd93e0f6d';
+
+    protected static $errorNames = [self::PWNED_ERROR => 'PWNED_ERROR'];
+
+    public $message = 'This password has been leaked in a data breach, it must not be used. Please use another password.';
```

Comment:
should be added in `validators.en.xlf` + any language you know :) 

not sure we'll do another round of good first issues for the remaining locales :}